Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM rasa/rasa:2.7.1-spacy-en AS trained
+
+WORKDIR /hotel-bot
+COPY . /hotel-bot
+
+RUN rasa train
+
+FROM rasa/rasa:2.7.1-spacy-en
```

Comment:
Any reason why you have broken this down into two separate steps? As in, why use an intermediate image?